client/src/components/_common/Pill/Pill.module.scss

```diff
@@ -1,11 +1,13 @@
-.pill {
-  padding: 0 1em; /* buffer retained on 2+ digit badge, without increase on 1 digit badge */
-  min-width: 1.3em;  /* ~18px design * 1.2 design-to-app ratio (rounded) */
+/* TODO: Consider sharing code between (History)Badge and Pill */
+.root {
+  font-size: 0.75rem; /* 12px (10px design * 1.2 design-to-app ratio) */
+  font-weight: 500;
+  padding: 0.2em 0.5em; /* aim for ~23px height (19px design * 1.2 design-to-app ratio) */
```

Comment:
The change from `1em` to `0.5em` is because only [Dashboard](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/specs/) has pills so _horizontally "puffy"_, and whether that should be from `padding` or `min-width` is unclear.